stratum/src/lib.rs

```diff
@@ -364,15 +364,30 @@ mod tests {
         result
     }
 
+    fn get_available_test_addr(start: u32, end: u32) -> SocketAddr {
+        let addr = (start..end)
+            .map(|port| SocketAddr::from_str(&format!("127.0.0.1:{}", port)).unwrap())
+            .find(|addr| match TcpListener::bind(addr) {
+                Ok(v) => {
+                    drop(v);
+                    true
+                }
+                Err(_) => false,
+            })
```

Comment:
How about `TcpListener::bind(addr).is_ok()`?
I think socket will be automatically dropped after this statement.

Reply:
This code is redundant. I'll fix this. I was trying to fix the port problem described above, because I think the port is still bound.